packages/lib/package.json

```diff
@@ -19,6 +19,7 @@
   },
   "dependencies": {
     "@leafygreen-ui/emotion": "^1.0.2",
+    "facepaint": "^1.2.1",
```

Comment:
I wouldn't normally want to include this by default in every package however:

- This package itself is very small
- We'll definitely be using it in other components

Reply:
Kk, should I leave or move to Modal alone?